docs/application/dotnet/guides/multimedia/media-controller.md

```diff
@@ -131,20 +132,20 @@ controller.PlaylistUpdated += (s, e) =>
 
 To send a command from the client and to process it on the server side, follow these steps:
 
-1.  To send a command on the client side, use the `SendPlaybackCommand()` method of the [Tizen.Multimedia.Remoting.MediaController](https://samsung.github.io/TizenFX/latest/api/Tizen.Multimedia.Remoting.MediaController.html) class:
+1.  To send a command on the client side, use the `RequestAsync()` method of the [Tizen.Multimedia.Remoting.MediaController](https://samsung.github.io/TizenFX/latest/api/Tizen.Multimedia.Remoting.MediaController.html) class:
 
     ```csharp
-    controller.SendPlaybackCommand(MediaControlPlaybackCommand.Pause);
+    mediaController.RequestAsync(new PlaybackCommand(MediaControlPlaybackCommand.Play));
     ```
 
 2. To process the received command on the server side, add an event handler to the `PlaybackCommandReceived` event of the [Tizen.Multimedia.Remoting.MediaControlServer](https://samsung.github.io/TizenFX/latest/api/Tizen.Multimedia.Remoting.MediaControlServer.html) class:
```

Comment:
Isn't this `PlaybackCommandReceived` also should updated to `PlaybackActionCommandReceived`?